wfirst-sit-debug/wfirst-sit-dev-environment.yml

```diff
@@ -0,0 +1,60 @@
+name: wfirst-sit-dev
+channels:
+- http://ssb.stsci.edu/astroconda
+- defaults
+dependencies:
+# Python
+- python==3.7.5
+# Environment support packages
+- ipykernel==4.8.2
+# Masterclass packages
+- astropy==3.2.2
+- astroscrappy==1.0.5
+- cfitsio==3.440
+- cubeviz==0.3.0
+- ds9==7.4
+- dust_extinction==0.7
+- fftw==3.3.4
+- fitsblender==0.3.6
+- fitsverify==4.18
+- ginga==3.0.1
+- glue-core==0.15.6
+- glue-vispy-viewers==0.12.2
+- gwcs==0.11.0
+- jplephem==2.9
+- jwst-visit-parser==0.1.2
+- jwst_footprints==2.2.0
+- jwst_gtvt==0.1.0
+- jwxml==0.3.0
+- matplotlib==3.1.1
+- numpy==1.14.6
+- pandas==0.25.2
+- pandokia==2.4.1
+- photutils==0.7.1
+- poppy==0.8.0
```

Comment:
poppy and webbpsf should be updated to version 0.9.0 to work with astropy 4.0.0.  This version is incompatible.